### ISSUE TITLE
API Docs for Commands

### ISSUE THREAD

**Issue by mariotacke** [Sonarr/Sonarr#5416]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

For the longest time, I could invoke `/api/command/` with

```
{
  "name": "DownloadedEpisodeScan",
  "path": "<my import path>"
  "importMode": "move"
}
```

which would happily import episodes which were already matched/could be matched without user input.

This endpoint appears to no longer exists. I cannot find any commits removing it/feature replacing it or otherwise. It now returns 415 Method Not Allowed.

### Expected Behavior

I've used this endpoint for many years and it was triggered via curl after an episode completed downloading. I expect to invoke this Sonarr command which then scans the provided path, and perform the import as it would if triggered via the UI.

### Steps To Reproduce

```
curl -vvvvv https://my-sonarr/api/command -X POST --header "X-Api-Key:<api key>" -d '{"name":"DownloadedEpisodesScan","path":"/watch/Some.Episode.Folder/","importMode":"Move"}'
```

### Environment

```markdown
- OS: Ubuntu/docker
- Sonarr: 4.0.0.369
- Docker Install: Docker version 20.10.6, build 370c289
- Using Reverse Proxy: no
- Browser: curl
```

### What branch are you running?

Develop

### Trace Logs?

No trace logs are generated when hitting the endpoint. It appears to have been removed entirely.

### Anything else?

Looking at the UI network traffic, it appears to now require a two step process:

1. Invoke GET /api/v3/manualimport?folder=<folder>&filterExistingFiles=true
2. The frontend client then generates a POST payload and submits it to: POST /api/v3/command {"files":[{"...a fairly complex object to perform the import"}],importMode:"move","name":"ManualImport"}

Is there a way to auto-generate the "simple" DownloadedEpisodeScan behavior from before? Are all fields required in the "ManualImport" command?

**Comment by mariotacke** [comment]
Found this https://github.com/Sonarr/Sonarr/issues/3599#issuecomment-594105342 comment from March 3, 2020 which suggests the approach above.

**Comment by mariotacke** [comment]
Appending `--header "Content-Type: application/json; charset=utf-8"` seems to have remedied the issue (wasn't needed in the past). Still curious why this command is undocumented/no commits exists for it?

**Comment by markus101** [comment]
As you've discovered, nothing has happened to it. and you can see it in use via Manual Import's Move Automatically from Wanted: Missing. The change in behaviour is likely due to the new libraries being used in v4.

The individual commands aren't currently tracked in the API docs, the old v2 API were removed as they are woefully out of date. https://sonarr.tv/docs/api/

**Comment by mariotacke** [comment]
Thanks @markus101. Looking forward to the new docs documenting the commands eventually.
